src/pages/edge/edge-network/release-notes.md

```diff
@@ -8,6 +8,13 @@ keywords:
 
 # Release notes
 
+## October 16, 2024
+
+### iOS Edge 5.0.3
+
+* Adds 507 as recoverable error code and applies valid retry-after timeout values on next retry if provided on the response, otherwise applies default timeout.
+* This release depends on minimum AEPCore v5.3.1.

```

Comment:
We have never included dependency information in the release notes, as it is already available in the Podspec and Package.swift files